gradle/libs.versions.toml

```diff
@@ -108,8 +108,8 @@ tink = { module = "com.google.crypto.tink:tink-android", version.ref = "tink" }
 
 coroutines-test = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-test", version.ref = "coroutines" }
 
-kotlin-bom = "org.jetbrains.kotlin:kotlin-bom:1.8.0"
-compose-bom = "androidx.compose:compose-bom:2023.05.01"
+kotlin-bom = "org.jetbrains.kotlin:kotlin-bom:1.9.10"
+compose-bom = "androidx.compose:compose-bom:2023.10.01"
```

Comment:
Unfortunately this is also used by the debug view, so it might be a breaking change there. Not sure how many people are using the debug view in android though..., specially people stil using kotlin 1.6 🤔

Reply:
Can we separate it?
Wanna take over? 😇

Reply:
I tried it for a bit today. I was running into a bunch of issues since we are trying to use multiple kotlin versions in the project before I had to leave for a meeting. I can take over though. Might not have time to get it done till next week though